tests/e2e_regression/damask/expected/emerald_contract_account.body

```diff
@@ -1,6 +1,12 @@
 {
   "address": "oasis1qz2rynvcmrkwd57v00298uc2vtzgatde3cjpy72f",
-  "balances": [],
+  "balances": [
+    {
+      "balance": "0",
+      "token_decimals": 0,
+      "token_symbol": ""
```

Comment:
we don't index any emerald blocks in the damask e2e range; nexus defaults to returning a 0 balance if no blocks have been indexed. (in order to be consistent with nexus' internal state)